src/sass/messaging/partials/_message-compose.scss

```diff
@@ -288,6 +286,17 @@ button.msg-btn-save {
     top: 0;
     width: auto;
   }
+
+  @media (max-width: $medium-screen) {
+    -webkit-flex: 0 0;
```

Comment:
I think this might be more of a question for core, but I saw that `autoprefixer` and `postcss` were both in the package.json, though I didn't see it configured in our webpack config.

If they are set up, they should automatically apply vendor prefixes for us so that we don't miss any.

https://github.com/postcss/postcss-loader